e2e/04-username-payments-flow.e2e.spec.ts

```diff
@@ -0,0 +1,79 @@
+import { i18nObject } from "../app/i18n/i18n-util"
+import { loadLocale } from "../app/i18n/i18n-util.sync"
+import { selector, enter } from "./utils"
+import { getInvoice } from "./utils/graphql"
+
+describe("Username Payment Flow", async () => {
+  loadLocale("en")
+  const LL = i18nObject("en")
+  const timeout = 30000
+  const username: string = "extheo"
```

Comment:
@nicolasburtey I couldn't create another test account because I don't have a phone number to use for it. How I can I go about this?
But temporarily this works.

Reply:
> @nicolasburtey I couldn't create another test account because I don't have a phone number to use for it. How I can I go about this? But temporarily this works.

you can sync with anyone on the backend/infra they can get you onboarded